src/AzureStack/Admin/ComputeAdmin/Compute.Admin/Microsoft.AzureStack.Management.Compute.Admin.csproj

```diff
@@ -7,13 +7,14 @@
     <PropertyGroup>
         <PackageId>Microsoft.AzureStack.Management.Compute.Admin</PackageId>
         <Description>Provides developers with libraries for the compute admin under AzureStack to manage compute resources.</Description>
-        <VersionPrefix>0.2.0-preview</VersionPrefix>
+        <VersionPrefix>0.3.0-preview</VersionPrefix>
         <AssemblyName>Microsoft.AzureStack.Management.Compute.Admin</AssemblyName>
         <PackageTags>Microsoft Azure Stack;Compute;REST HTTP client;azureostackfficial;netcore451511</PackageTags>
```

Comment:
You might want to add 'Managed Disks' to the PackageTags.  Add others if you feel necessary.

Reply:
Sure.